cmd/status/print/print.go

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 The Kubernetes Authors.
+// SPDX-License-Identifier: Apache-2.0
+
+package print
+
+// Printer defines the interface the status command expects
+// for outputting information about status of resources. Different
+// implementations allow output formats tailored to different use cases.
```

Comment:
When defining an interface, its odd to call out who's expected to use it.

Reply:
I have updated the comment.